### DISCUSSION TITLE
2023.11.30 第十八讲 回归常态 致敬旧时代 读书讲故事

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#41] [General]
https://www.clubhouse.com/room/kgpcl5VK1ZTRBgB6Z:3zSNjws_AMKO7yDcrjhZWhiHct2SGjlt1TutEQq_JVQ?utm_medium=ch_instant_invite&utm_campaign=lpYfcJN9MC3YSlNJ-t3K_A-1004672

**Comment by guttasingapore** [comment]
致敬查理芒格 Charlie Munger （1924-2023）

-Poor Charlie's Almanack: The Wit and Wisdom of Charles T. Munger(2005)
《穷查理的普通常识-查理．芒格的人生哲学》
https://36kr.com/p/1722004389889
如何拥有多元思维模型 -  大约八十至九十个这样的模型就足以解决90%的问题
https://www.mbachina.com/html/ckgsb/201909/199907.html
SEEKING WISDOM From Darwin to Munger By Peter Bevelin （请慎读）
https://www.madewill.com/wp-content/uploads/2018/03/探索智慧：从达尔文到查理芒格.pdf
芒格建议小结 https://m.gelonghui.com/p/1181

Q: 如何寻找、形成 芒格巴菲特类似的合伙人关系 （40-99）， 三观一致的灵魂伴侣？

ref: 被芒格认同的李录
https://assets-global.website-files.com/5ef3c7300432b40ed865991a/5f0937a6d3ce597c81f03bce_%E6%8E%A8%E8%8D%90%E9%98%85%E8%AF%BB%204.30.pdf

**Reply by Lewisrxliu** [reply]
感谢三叔推荐这本书，下次来主讲论革命-汉娜·阿伦特 Hannah Arendt
https://en.wikipedia.org/wiki/On_Revolution
https://books.google.com/books/about/On_Revolution.html?id=CPnkAgAAQBAJ
参考三叔主讲精读过的Love in the Time of Cholera, 霍乱时期的爱情

**Reply by guttasingapore** [reply]
回应霍乱时期的爱情： 
刘震云评中国文学史上绝顶绝伦的那种诗
-君问归期未有期，巴山夜雨涨秋池。何当共剪西窗烛，却话巴山夜雨时。《夜雨寄北》唐-李商隐 
哲学停止的地方，文学出现了- 刘震云
https://www.chinanews.com.cn/cul/2023/04-07/9985530.shtml

**Comment by guttasingapore** [comment]
> 致敬查理芒格 Charlie Munger （1924-2023）
> 
> -Poor Charlie's Almanack: The Wit and Wisdom of Charles T. Munger(2005) 《穷查理的普通常识-查理．芒格的人生哲学》 https://36kr.com/p/1722004389889 如何拥有多元思维模型 - 大约八十至九十个这样的模型就足以解决90%的问题 https://www.mbachina.com/html/ckgsb/201909/199907.html SEEKING WISDOM From Darwin to Munger By Peter Bevelin （请慎读） [https://www.madewill.com/wp-content/uploads/2018/03/探索智慧：从达尔文到查理芒格.pdf](https://www.madewill.com/wp-content/uploads/2018/03/%E6%8E%A2%E7%B4%A2%E6%99%BA%E6%85%A7%EF%BC%9A%E4%BB%8E%E8%BE%BE%E5%B0%94%E6%96%87%E5%88%B0%E6%9F%A5%E7%90%86%E8%8A%92%E6%A0%BC.pdf)
> 
> 如何寻找、形成 芒格巴菲特类似的合伙人关系 （40-99）， 三观一致的灵魂伴侣？
> 
> 被芒格认同的李录 
李录谈现代化 （2014）- 回答了部分今天同学们提出的好问题
https://assets-global.website-files.com/5ef3c7300432b40ed865991a/5f0935ee7646e6ff74f6d311_李录谈现代化%20（全文）大字号.pdf
李录书单 https://assets-global.website-files.com/5ef3c7300432b40ed865991a/5f0937a6d3ce597c81f03bce_%E6%8E%A8%E8%8D%90%E9%98%85%E8%AF%BB%204.30.pdf

https://zh.wikipedia.org/zh-sg/为什么现在统治世界的是西方
Morris 定律（Morris Theorem） “Change is caused by lazy, greedy, frightened people looking for easier, more profitable,
and safer ways to do things. And they rarely know what they are doing.”
“历史，就是懒惰、贪婪、又充满恐惧的人类，在寻求让生活更容易、安全、有效的方式时创造的，而人类对此毫无意识。”
Why The West Rules - For Now: The Patterns of History and what they reveal about the Future, Ian Morris 
《西方将主宰多久──从历史的发展模式看世界的未来》
https://books.google.com.sg/books/about/Why_The_West_Rules_For_Now.html?id=bMi1XVXAl48C&redir_esc=y

**Reply by ergatea** [reply]
# 追念恩师查理·芒格
李录
2023 年 11 月 30 日
周二在亚洲出差，接的查理家人的电话，告知查理已经生命垂危。马上搭飞机返美，临行
前通过查理女儿的电话，和他做最后的通话。此时查理已经意识模糊，但是中间几次都能听到查
理努力的发出声音，仿佛听到了我的话。下飞机时知道，查理已经在几个小时之前离世了…
来到查理在 Santa Barbara 的家里，和查理家人后代回忆起他生前的点点滴滴。我又回到查
理的图书室，整整二十年前，就是在这间屋子里，也是感恩节周末的下午，我们交谈了好几个小
时，开启了我们此后二十年的投资伙伴关系，查理也从此成为我终生的良师益友，人生楷模。
查理家人不做遗体告别，但是为了我特意在第二天中午做了这项安排，我万分感激。
在教堂的一角，查理静静的闭目躺在床上，和离世前一模一样，安详、宁静，脸上流露出
一丝不易察觉的微笑，一时间竟让我想起在泰国清迈寺庙里看到过的肉身佛。查理经由终生修
行，也如得道高僧，金身不腐，浑身都散发出永恒的光芒。查理和家人都不是佛教徒，当然不会
成为肉身佛，但是他的精神遗产将世代相传。
中国文化的现代化必然离不开市场经济、商业社会环境下士大夫精神的现代演化。芒格先
生是现代士大夫的化身。以诚实、德行、勤勉、修炼自我；凭借智慧、真知，以互惠共赢的方式
致富；借凭财富自由追求人格独立、达世济人；以终生学习、独立思想、理性诚实追求真知；以
榜样的力量教诲后人，传播思想。此乃商才士魂，当代士人典范！
https://assets-global.website-files.com/5ef3c7300432b40ed865991a/656a90a8013607f02556e6bf_%E8%BF%BD%E5%BF%B5%E6%81%A9%E5%B8%88%E6%9F%A5%E7%90%86%E8%8A%92%E6%A0%BC.pdf
